playwright-e2e/settings/userCredentials.ts

```diff
@@ -1,6 +1,16 @@
-const e2ePw = process.env.E2E_TEST_PASSWORD || "";
+const e2ePw = process.env.E2E_TEST_PASSWORD || '';
 
 export const newSwanseaLocalAuthorityUserOne = {
-  email: "local-authority-swansea-0001@maildrop.cc",
+  email: 'local-authority-swansea-0001@maildrop.cc',
   password: e2ePw,
 };
+export const systemUpdateUser ={
+  email: 'fpl-system-update@mailnesia.com',
+  password: e2ePw,
+
+};
+export const swanseaUser ={
```

Comment:
Should we continue to be using "kurt" account? As it has performance issues.

Reply:
not using,I added for local testing.